### PR TITLE
Feature for issue #244: Attempt at configuring the long press time

### PR DESCRIPTION
**Pull request by fras2560**
The default is 251 since I believe that is the default of hammer.js
Added it to the documentation
I had to set it at Core and ItemSet although there is probably a better way to achieve this.

### CODE REVIEW

**Comment by Thomaash** [comment]
Hi @fras2560,

I think that @yotamberk is the best person to review this. I generally only participate in Timeline when it's about the build process or the projects I maintain (like Vis Util and Vis Data in #243).

File: lib/timeline/Core.js [hunk 1]
```diff
@@ -1044,6 +1045,11 @@ class Core {
       direction: contentsOverflow ? Hammer.DIRECTION_ALL : Hammer.DIRECTION_HORIZONTAL
     });
 
+    // set the long press time
+    this.hammer.get('press').set({
+      time: this.options.multiselectLongPressTime
```
Comment (yotamberk) [inline lib/timeline/Core.js]: I think you're missing default value here.

**Comment by yotamberk** [comment]
@fras2560 Thanks for the PR but I think the name of the variable is not correct. I think it should be simply `longSelectPressTime` since it is not per-say only for multiselect

File: lib/timeline/Core.js (hunk 1 above)
Comment (fras2560) [reply lib/timeline/Core.js]: Ok, where should I define the default value?

Comment (fras2560) [reply lib/timeline/Core.js]: The default value should now be set.

**Pull request merged**